lua/kickstart/plugins/debug.lua

```diff
@@ -24,28 +24,59 @@ return {
     -- Add your own debuggers here
     'leoluz/nvim-dap-go',
   },
-  keys = function(_, keys)
-    local dap = require 'dap'
-    local dapui = require 'dapui'
-    return {
-      -- Basic debugging keymaps, feel free to change to your liking!
-      { '<F5>', dap.continue, desc = 'Debug: Start/Continue' },
-      { '<F1>', dap.step_into, desc = 'Debug: Step Into' },
-      { '<F2>', dap.step_over, desc = 'Debug: Step Over' },
-      { '<F3>', dap.step_out, desc = 'Debug: Step Out' },
-      { '<leader>b', dap.toggle_breakpoint, desc = 'Debug: Toggle Breakpoint' },
-      {
-        '<leader>B',
-        function()
-          dap.set_breakpoint(vim.fn.input 'Breakpoint condition: ')
-        end,
-        desc = 'Debug: Set Breakpoint',
-      },
-      -- Toggle to see last session result. Without this, you can't see session output in case of unhandled exception.
-      { '<F7>', dapui.toggle, desc = 'Debug: See last session result.' },
-      unpack(keys),
-    }
-  end,
+  keys = {
+    -- Basic debugging keymaps, feel free to change to your liking!
+    {
+      '<F5>',
+      function()
+        require('dap').continue()
+      end,
+      desc = 'Debug: Start/Continue',
+    },
+    {
+      '<F1>',
+      function()
+        require('dap').step_into()
+      end,
+      desc = 'Debug: Step Into',
+    },
+    {
+      '<F2>',
+      function()
+        require('dap').step_over()
+      end,
+      desc = 'Debug: Step Over',
+    },
+    {
+      '<F3>',
+      function()
+        require('dap').step_out()
+      end,
+      desc = 'Debug: Step Out',
+    },
+    {
+      '<leader>b',
+      function()
+        require('dap').toggle_breakpoint()
+      end,
+      desc = 'Debug: Toggle Breakpoint',
+    },
```

Comment:
I would like to suggest that we try this approach:
```
    { '<F5>', <cmd>DapContinue<CR>, desc = 'Debug: Start/Continue' },
    { '<F1>', <cmd>DapStepInto<CR>, desc = 'Debug: Step Into' },
    { '<F2>', <cmd>DapStepOver<CR>, desc = 'Debug: Step Over' },
    { '<F3>', <cmd>DapStepOut<CR>, desc = 'Debug: Step Out' },
    { '<leader>b', <cmd>DapToggleBreakpoint<CR>, desc = 'Debug: Toggle Breakpoint' },
```

Reply:
Ok I have tested this and this works ok, but it is missing the `<F7>` keybind and I dont know the command for that

Reply:
Unfortunately, there does not seem to be a user command defined for them at this time.
There are several ways to do like:
* Create a user command
```
vim.api.nvim_create_user_command('DapUiToggle', function()
  require('dapui').toggle()
end, { nargs = 0 })
return {
  ...
    { '<F7>', '<cmd>DapUiToggle<CR>', desc = 'Debug: See last session result.' },
  ...
}
```

* Write as a lua function
```
return {
  ...
    { '<F7>', '<cmd>lua require("dapui").toggle()<CR>', desc = 'Debug: See last session result.' },
  ...
}
```

* Keep the status quo